tests/10apidoc/31room-state.pl

```diff
@@ -64,7 +64,7 @@
       )->then( sub {
          my ( $body ) = @_;
 
-         assert_json_keys( $body, qw( sender event_id room_id content ));
```

Comment:
Note that this is when we've specified ?format=event, i.e. asking for the raw federation format